maintainer/conda/MDAnalysis/meta.yaml

```diff
@@ -0,0 +1,76 @@
+package:
+  name: mdanalysis
+  # This has to be changed after a release
+  version: "0.14.0"
```

Comment:
@richardjgowers are you still using the `maintainer/change_release.sh` script? If yes, then changing this release should also be hacked into the script.